content/docs/guides/contribute/code-review.fr.md

```diff
@@ -17,4 +17,24 @@ Nous vous soumettons quelques conseils et recommandations qui nous semblent pert
 
 ## La pyramide de la revue de code
 
-{{< figure src="/images/docs/contribute/code_review_pyramid.svg" link="https://www.morling.dev/blog/the-code-review-pyramid/">}}
\ No newline at end of file
+{{< figure src="/images/docs/contribute/code_review_pyramid.svg" link="https://www.morling.dev/blog/the-code-review-pyramid/">}}
+
+## Script pour la Revue des PRs
+
+Lors de la revue d'une PR, il est utile de tester les changements en démarrant une instance de l'application OSRD basée sur la branche de la PR.
+
+Pour simplifier ce processus, nous avons créé un script qui lance automatiquement une instance dédiée de l'application en utilisant le numéro de la PR. Le script récupère les images Docker générées par la CI et lance l'application, chaque conteneur fonctionnant sur des ports différents de ceux de l'environnement de développement. Cela permet d'exécuter les deux instances en parallèle, sans conflits (idéal pour comparer les modifications, par exemple). L'avantage principal est que les revues sont plus rapides, car il n’est pas nécessaire de recompiler le code ; il suffit de télécharger les images déjà construites.
+
+De plus, vous pouvez fournir une sauvegarde de l'infrastructure, que le script chargera directement dans l'application.
```

Comment:
The PG dump contains all the data (studies, train schedules, rolling-stock, electrical profiles, etc.) not only the infrastructure:
```suggestion
De plus, vous pouvez fournir une sauvegarde de la base de données, que le script chargera directement dans l'application.
```